### ISSUE TITLE
The wizard bar in the RA should display the correct amount of steps with the correct increments on each step in the vetting process.

### ISSUE THREAD

**Issue by phavekes** [OpenConext/Stepup-RA#346]
[This issue is imported from pivotal - Originaly created at Dec 12, 2014 by Daan van Renterghem](https://www.pivotaltracker.com/story/show/84447362)

may be a duplicate of https://www.pivotaltracker.com/story/show/84122914

**Comment by phavekes** [comment]
Apart from the active label states (#84122914) and a missing vetting success screen with progress 100% (#84379036), the steps seem fine. Is there a concrete complaint? Perhaps I should ask Eefje and/or wait for beta feedback. (Reinier Kip - Dec 12, 2014)

**Comment by phavekes** [comment]
If the other two issues resolve this, then indeed the ticket is a duplicate. I was unsure whether or not there were 4 steps in the RA and the wizard and if the bar was updating properly, but only after making the issues I saw the other issue which has a slightly different text, hence left it for you to decide ;)
 (Daan van Renterghem - Dec 12, 2014)
